lib/active_model/serializable_resource.rb

```diff
@@ -1,7 +1,7 @@
 require 'set'
 module ActiveModel
   class SerializableResource
-    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links])
+    ADAPTER_OPTION_KEYS = Set.new([:include, :fields, :adapter, :meta, :meta_key, :links, :batch_cache])
```

Comment:
I'm pretty sure this is wrong, because the options passed to the adapter are passed by the renderer, but maybe I'm forgetting something.  I believe the below is how it would be used.

```ruby
ActiveModel::SerializableResource.new(@post, adapter: :json_api).as_json(batch_cache: { disable: true})
```